src/apps/pillarx-app/components/WalletPortfolioButtons/WalletPortfolioButtons.tsx

```
@@ -38,6 +41,94 @@ const WalletPortfolioButtons = () => {
+      );
+
+      const data = await response.json();
+      console.log('Coinbase API response for onramp session:', data);
+
+      console.log('Onramp session created successfully');
+
+      const onrampUrl = data?.session?.onrampUrl;
+
+      if (!onrampUrl) {
+        throw new Error('No URL received from Coinbase API');
+        return;
```

Biome (2.1.2)

[error] This code is unreachable



(lint/correctness/noUnreachable)